src/material-experimental/mdc-chips/chip-row.html

```diff
@@ -1,19 +1,32 @@
-<span class="mdc-chip__ripple"></span>
+<ng-container *ngIf="!_isEditing()">
+  <span class="mdc-chip__ripple"></span>
 
-<span matRipple class="mat-mdc-chip-ripple"
-     [matRippleAnimation]="_rippleAnimation"
-     [matRippleDisabled]="_isRippleDisabled()"
-     [matRippleCentered]="_isRippleCentered"
-     [matRippleTrigger]="_elementRef.nativeElement"></span>
+  <span matRipple class="mat-mdc-chip-ripple"
+       [matRippleAnimation]="_rippleAnimation"
+       [matRippleDisabled]="_isRippleDisabled()"
+       [matRippleCentered]="_isRippleCentered"
+       [matRippleTrigger]="_elementRef.nativeElement"></span>
+</ng-container>
 
-<div role="gridcell">
-  <div #chipContent tabindex="-1"
-       class="mat-chip-row-focusable-text-content mat-mdc-focus-indicator">
-  	 <ng-content select="mat-chip-avatar, [matChipAvatar]"></ng-content>
-  	 <span class="mdc-chip__text"><ng-content></ng-content></span>
-  	 <ng-content select="mat-chip-trailing-icon,[matChipTrailingIcon]"></ng-content>
+<div class="mat-chip-content">
+  <div role="gridcell">
+    <div #chipContent tabindex="-1"
+         class="mat-chip-row-focusable-text-content mat-mdc-focus-indicator mdc-chip__primary-action"
```

Comment:
I see this was already like this, but should also be `mat-mdc-`

Reply:
Done (assuming you meant mat-chip-row-focusable-text-content)